lib/src/gantt_default_week_header.dart

```diff
@@ -7,6 +7,17 @@ class GanttChartDefaultWeekHeader extends StatelessWidget {
   }) : super(key: key);
   final DateTime weekDate;
 
+  int get weekNumberWithinThisYear {
+    final numberWeek = (DateTime.utc(
+                    weekDate.year, weekDate.month, weekDate.day)
+                .difference(DateTime.utc(weekDate.year, 1, 1, 0, 0, 0, 0, 0))
+                .inDays /
+            7)
+        .ceil();
+
+    return numberWeek;
+  }
```

Comment:
I found this library using the correct iso calculations
https://github.com/KadaDev/week_of_year/blob/master/lib/date_week_extensions.dart

Reply:
Impressive, thanks for sharing this